<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionlivraison</class>
 <widget class="QDialog" name="gestionlivraison">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestion Livraison</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Livraison</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QLineEdit:hover
{
border: 2px solid rgb(82, 124, 124)
}

</string>
     </property>
     <property name="placeholderText">
      <string>Type Here ...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat Livraison</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QLineEdit:hover
{
border: 2px solid rgb(82, 124, 124)
}

</string>
     </property>
     <property name="placeholderText">
      <string>Type Here ...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Num Livreur</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QLineEdit:hover
{
border: 2px solid rgb(82, 124, 124)
}

</string>
     </property>
     <property name="placeholderText">
      <string>Type Here ...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>170</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QDateEdit
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QDateEdit:hover
{
border: 2px solid rgb(82, 124, 124)
}

</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>360</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}
QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}



</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Commande</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QLineEdit:hover
{
border: 2px solid rgb(82, 124, 124)
}

</string>
     </property>
     <property name="placeholderText">
      <string>Type Here ...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>220</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QLineEdit:hover
{
border: 2px solid rgb(82, 124, 124)
}

</string>
     </property>
     <property name="placeholderText">
      <string>Type Here ...</string>
     </property>
    </widget>
    <widget class="QTableView" name="tab_livraison">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>501</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableView
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QTableView:hover
{
border: 2px solid rgb(82, 124, 124)
}
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tttt">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>320</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QLineEdit:hover
{
border: 2px solid rgb(82, 124, 124)
}
</string>
     </property>
     <property name="placeholderText">
      <string>Type Here ...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chercherid">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>320</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}
</string>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trierid">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>390</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}</string>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>480</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Livraison</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>470</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QLineEdit:hover
{
border: 2px solid rgb(82, 124, 124)
}
</string>
     </property>
     <property name="placeholderText">
      <string>Type Here ...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>470</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}



</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generer">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}
</string>
     </property>
     <property name="text">
      <string>Generer PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Francais">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>20</y>
       <width>56</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}



</string>
     </property>
     <property name="text">
      <string>Fr</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Anglais">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>50</y>
       <width>56</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}



</string>
     </property>
     <property name="text">
      <string>An</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Excel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}



</string>
     </property>
     <property name="text">
      <string>Generer Excel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>360</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}

</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>330</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Chercher par : </string>
     </property>
    </widget>
    <widget class="QPushButton" name="cherchedate">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>320</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}
</string>
     </property>
     <property name="text">
      <string>DATE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cherchernum">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>320</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}
</string>
     </property>
     <property name="text">
      <string>NUM LIVREUR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>400</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier par :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trierdate">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>390</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}
</string>
     </property>
     <property name="text">
      <string>DATE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="triernum">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>390</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 2px solid rgb(0, 0, 0);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right: 20px;
background-color: rgb(114, 171, 171);
}

QPushButton:hover
{
border: 2px solid rgb(82, 124, 124)
}
</string>
     </property>
     <property name="text">
      <string>NUM LIVREUR</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
